README.md

```diff
@@ -15,9 +15,9 @@ You can either use your own image, or one of the ones supplied in the [dockerfil
 For instance, for Dendrite:
 ```
 # build a docker image for Dendrite...
-(cd dockerfiles && docker build -t complement-dendrite -f Dendrite.Dockerfile .)
+$ cd dockerfiles && docker build -t complement-dendrite -f Dendrite.Dockerfile .
```

Comment:
afaik this didn't need `()`'s

Reply:
It doesn't but the subsequent command has to be run from the parent directory. Removing the parentheses breaks the next command.

Reply:
Oh, nice trick!